fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -689,6 +689,19 @@ struct DDQueue : public IDDRelocationQueue {
 
 	int moveReusePhysicalShard;
 	int moveCreateNewPhysicalShard;
+	enum NewPhysicalShardReason {
```

Comment:
Nit, maybe consider renaming this, since it seems that this counts not only the reasons of creating a new physical shard, but also reasons of retrying selecting the destination team for a data move in general.

Reply:
Are you saying that it is possible that these reason may not result in a new physical shard created?

Reply:
Yes, also put offline discussion here: this reveals a potential performance issue for shard selection.